mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -235,6 +252,17 @@ public override void Run (IEnumerable<string> args)
                                   })
                                   .Where (f => Directory.Exists (f.Path));
 
+                oldFrameworkXmls = fxconfig.Root
```

Comment:
Does this feature work at all if you don't use it with a `frameworks` enabled set of documentation? If it does not, please add a check to ensure that frameworks is enabled, and otherwise write a warning to output saying that you had to disable it because it only works with frameworks mode.

Reply:
to clarify, what I mean by "no frameworks mode", is just running mdoc without a `frameworks.xml` file, and without the `-fx` parameter. So for example:

```
mdoc update someassembly.dll -o outxml -statistics stats.txt
```

What will happen with the above invocation ... ideally, it would simply work, but if that is not possible (since you're using the framework index files), then at a minimum it should write a warning and make sure that it's disabled so it doesn't fail.

Reply:
This feature works in non-framework mode. In this case, all types are processed with EmptyFrameworkEntry, so we treat it as a usual framework and generate something like this:
```
Framework: Empty
--------
Types Added: 0
Types Removed: 1
Types Total: 3

Namespaces Added: 0
Namespaces Removed: 0
Namespaces Total: 1

Members Added: 1
Members Removed: 6
Members Total: 10
```